pocket_lab/pl_gui_views.c

```diff
@@ -177,6 +193,15 @@ static adi_fft_data_to_volt_conv data_to_volt_wrt_vref;
  * binary data */
 static adi_fft_code_to_straight_bin_conv code_to_straight_binary;
 
+/* Pocket Lab Instance */
+static struct pl_gui_init_param pl_instance =  {
+	.event = NULL,
+	.event1 = NULL
```

Comment:
what is this event1?

Reply:
These are the events we need for the application layer. The events are when stop is pressed / when 2048 samples are complete. We don't want the reason why these events were added and exposed in the library as it was for a specific cbm use case. Hence, I have kept a very generic name.

Reply:
Do you think it makes more sense with event1, event2 as event names?